controlpanel/frontend/urls.py

```diff
@@ -53,5 +53,6 @@
     path("webapps/<int:pk>/datasource-access/", views.GrantAppAccess.as_view(), name="grant-app-access"),
     path("webapp-datasource-access/<int:pk>/", views.UpdateAppAccess.as_view(), name="update-app-access"),
     path("webapp-datasource-access/<int:pk>/delete/", views.RevokeAppAccess.as_view(), name="revoke-app-access"),
+    path("reset/", views.Reset.as_view(), name="home-reset"),
```

Comment:
again (sorry!) I'd be more explicit in the URL, something like `/user-home-reset/`.